.ci/test.sh

```diff
@@ -52,7 +52,7 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "lint" ]]; then
             "r-lintr>=2.0"
     pip install --user cpplint
     echo "Linting Python code"
-    pycodestyle --ignore=E501,W503 --exclude=./compute,./eigen,./.nuget,./external_libs . || exit -1
+    pycodestyle --ignore=E402,E501,W503 --exclude=./compute,./eigen,./.nuget,./external_libs . || exit -1
```

Comment:
Why was it needed? I can't find any new imports other that ones at the top of modules... Am I missing something?

Reply:
Ah, this was for test_dask. The check for whether or not the test is running on Linux system occurs before the remaining imports.

Reply:
Hmmm, strange... We have this Linux-test line in the `master` branch and there are no warnings.
https://github.com/microsoft/LightGBM/blob/f2695daba2506d047831e39aeb521e94adec5e8d/tests/python_package_test/test_dask.py#L4-L11

Reply:
weird! Huh. I'll remove E402 and see how the lint testing goes during test builds today.